DiskDetective/index.html

```diff
@@ -0,0 +1,91 @@
+<!doctype html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7" lang=""> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8" lang=""> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9" lang=""> <![endif]-->
+<!--[if gt IE 8]><!--> <html class="no-js" lang=""> <!--<![endif]-->
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+        <title>Disk Detective: Archive</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href='https://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
+        <link rel="stylesheet" href="css/bootstrap.min.css">
+        <style>
+            body {
+                padding-top: 50px;
+                padding-bottom: 20px;
+            }
+        </style>
+        <link rel="stylesheet" href="css/bootstrap-theme.min.css">
+        <link rel="stylesheet" href="css/main.css">
+        <link rel="icon" href="favicon.ico" type="image/ico"/>
+        <script src="js/vendor/modernizr-2.8.3-respond-1.4.2.min.js"></script>
+    </head>
+    <body onload="setAuthors(authors)">
+        <!--[if lt IE 8]>
+            <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
+        <![endif]-->
+      <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
+        <div class="container">
+          <div class="navbar-header">
+            <a class="navbar-brand headline" href="#/index">Archived Zooniverse Project: Disk Detective</a>
+            <a class="navbar-brand headline" href="#/authors">Authors</a>
+          </div>
+        </div>
+      </nav>
+
+    <!-- Main jumbotron for a primary marketing message or call to action -->
+    <section id="/authors">
+      <div class="jumbotron">
+        <div>
+          <h2>The people who make it all possible!</h2>
+          <p>
+              Thank you to the following folunteers, whose millions of clicks have
+              enabled us to produce our research.
+          </p>
+          <p id="author_list"></p>
+        </div>
+      </div>
+    </section>
+
+    <section id="/index">
+      <div class="jumbotron">
+        <div class="container container-flex">
+          <img src="img/disk.png" class="avatar cover"/>
+          <div>
+            <h1>Disk Detective</h1>
```

Comment:
This section should be at the first section on the page. At the moment, you have h2 headings ahead of h1.

Reply:
This is working like a single page app, so the other two sections are display none while this is rendered.

Reply:
That doesn't matter to a HTML parser, though. If you want this page to pass accessibility checks, and have decent SEO, H1 should be the first heading on the page.

Reply:
Ok, good to know 👍 . @wgranger this update should be made to any of the sites that have the extra static pages you've implemented as a SPA.

Reply:
Gotcha. Yeah, the h1 was put at the end so the css would display each section directly when targeted, (with #index appearing when no target present). I bet I can find another solution.